src/grammar.grammar

```diff
@@ -47,13 +47,13 @@ State{
 ConstraintsSection { "constraints" lineEnd ConstraintsBlock+ }
 
 ConstraintSign { "|" | "=>" | "&" | "==" | "<=>" | "!="}
-ConstraintsItem { Neg? ((identifier | string | id) | ImportFeature) | BracketItem }
+ConstraintsItem { Neg? ((identifier | string | id | extIdentifier) | ImportFeature) | BracketItem }
 BracketItem { OpenBracket ConstraintsItem CloseBracket }
 Neg { "!" }
 Brackets { !first OpenBracket Constraints CloseBracket }
 OpenBracket { "(" }
 CloseBracket { ")" }
-Number { number | (OpenBracket Number NumericOperator Number CloseBracket)}
+Number { ("-"? number) | (OpenBracket Number NumericOperator Number CloseBracket)}
```

Comment:
Switch to 
- UnsignedInteger for [0-9]+
- Integer for ([-]?)UnsignedInteger

Reply:
changed